python/pyspark/pandas/frame.py

```diff
@@ -10066,7 +10071,7 @@ def unstack(self) -> DataFrameOrSeries:
         )
 
     # TODO: axis, skipna, and many arguments should be implemented.
```

Comment:
Now we're only missing `level` and `**kwargs`.

Can we fix this to `# TODO: axis, skipna, level and **kwargs should be implemented.` while we're here ??

Reply:
Sounds good!